packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+| logo     | _slot_ | undefined | A logo/file type to add over the image preview           |
+
+### CardFooter
+
+This component has no custom props and only ensures the proper positioning of content and itself within the card.
+
+## Sample Code
+
+### Compact
+
+```jsx
+<Card orientation="horizontal">
+  <CardHeader image={<img src="powerpoint.png" />} title="Team offsite 2020" description="Onedrive > Files" />
+</Card>
+```
+
+### App
+
+```jsx
+<Card>
+  <CardHeader image={<img src="app_logo.png" />} title="App Name" description="Developer" />
+  <CardBody>
+    Donut chocolate bar oat cake. Dragée tiramisu lollipop bear claw. Marshmallow pastry jujubes toffee sugar plum.
+  </CardBody>
+</Card>
+```
+
+### Task
+
+```jsx
+<Card>
+  <CardHeader title="Alert in Teams when a new document is uploaded in channel" description="By Microsoft" />
+  <CardFooter>
+    <span>Automated</span>
+    <span>3290 uses</span>
+  </CardFooter>
+</Card>
+```
+
+### Image
+
+```jsx
+<Card>
+  <CardPreview src="monkey.png" alt="image of monkey" />
+  <CardBody>
+    <div>
+      <span>Length</span>
+      <span>4.6-39.6 IN</span>
+      <span>11.7-100.6 cm</span>
+    </div>
+    <div>
+      <span>Diet</span>
+      <span>Omnivore</span>
+      <span>nuts, fruits, eggs</span>
+    </div>
+  </CardBody>
+  <CardFooter>
+    <Button>Share</Button>
+  </CardFooter>
+</Card>
+```
+
+## Behaviours
+
+### Mouse
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the mouse](./assets/selectable-interaction-mouse.png)
+
+Clicking the card surface can toggle the checkbox state to `selected`. Note that checkboxes are composed together as an additional component and are not provided by default.
+
+1. Rest Default state is `unselected`
+2. Hover cursor style changes to `pointer`, background color and elevation follows interactive styling to show visual feedback
+3. Pressed cursor does not change background color and elevation follows interactive styling to show visual feedback for pressed state.
+4. MouseUp Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the mouse](./assets/non-interactive-interaction-mouse.png)
+
+Cards that have multiple actionable child components within them cannot be clickable. We preserve the use of `EnterKey` for moving focus to actionable controls inside the card, therefore the card surface cannot be used to set the state of the card but is reliant on the checkbox or similar inputs to set the card state.
+
+Mouse interaction
+
+- OnMouseEnter / Hover cursor style stays at default, rest state stays persistent no visual feedback is applied to the card surface
+- Hover & onClick (Checkbox) Hover state & pressed state applies to checkbox component
+- MouseUp (Checkbox) Card and Checkbox are both in selected state, clicking again toggles components to unselected state
```

Comment:
I think we should use click handler for this interaction, not mouse up. I am not sure if screen readers send mouse up/down events in all modes.

Reply:
Agreed, fixed